java/vector/src/main/codegen/templates/NullableValueVectors.java

```diff
@@ -105,6 +112,7 @@
     </#if>
     this.mutator = new Mutator();
     this.accessor = new Accessor();
+    <#if minor.class != "Int" && minor.class != "VarChar">
```

Comment:
I wonder is it easier to have a separate template for Legency Vectors instead of if statements?

Reply:
I am not sure about that. I am actually thinking if instead of having legacy vectors, can we just put all the new vectors in a different package. Changing the template code to generate legacy vectors and do delegations to new vectors is like a throwaway work because eventually we will get rid of template all-together.

Reply:
nvm, I have made the changes to template to delegate the ops for Nullable Int and Nullable VarChar. Building  things now locally and trying to run tests.